doc/man1/pbs_professional.7B

```diff
@@ -75,13 +75,13 @@ man page.
 .br
 .I PBS Professional Installation and Upgrade Guide
```

Comment:
Are we doing anything with this file? It links to pbsworks.com and the open confluence site. Also, are we not going to change the Installation and Upgrade guide?

Reply:
As of now, I am not sure about those @vstumpf and I have already asked @billnitzberg about those links and this file over email.

Reply:
we can address these later as well